spec/features/cart/checkout_spec.rb

```diff
@@ -95,6 +95,10 @@
       expect(page).to have_link("Checkout")
       expect(page).to have_link("Empty Cart")
 
+      within(".cart-#{@tire.id}") do
+        click_link "-"
+      end
+
       within(".cart-#{@tire.id}") do
         click_link "-"
```

Comment:
Needed to click the "-" button twice because once its at zero its not automatically removed from the cart. Once the quantity is at 0 and its pressed again its then removed. This can be refactored later but as of now its working !

Reply:
![](https://media3.giphy.com/media/100QWMdxQJzQC4/giphy.gif?cid=5a38a5a2b921c3f0123d82ba0945aa8c3929e9f3e2288caf&rid=giphy.gif)